repository-data/application/src/main/resources/hcm-config/configuration/translations/types.yaml

```diff
@@ -4,22 +4,22 @@ definitions:
       jcr:primaryType: hipposys:resourcebundles
       /en:
         jcr:primaryType: hipposys:resourcebundle
-        jcr:name: Content Cards
+        jcr:name: Navigation card
```

Comment:
Updated `hee:contentCards` CompoundType translation to `Navigation cards` (from `Navigation card`) so that `Navigation card` could be repurposed for `hee:ContentCard` DocumentType.

Fyi - Have changed caption of `Items` field on `hee:contentCards` CompoundType to `Navigation Card(s)` (to differentiate it from CompoundType name `Navigation Cards`)